### PR TITLE
[ENH] Integration testing for MRIQCWebAPI

### PR DESCRIPTION
**Pull request by oesteban**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -23,6 +26,11 @@ dependencies:
     - mkdir -p $SCRATCH && sudo setfacl -d -m group:ubuntu:rwx $SCRATCH && sudo setfacl -m group:ubuntu:rwx $SCRATCH
     - if [[ ! -d ~/data/${TEST_DATA_NAME} ]]; then wget --retry-connrefused --waitretry=5 --read-timeout=20 --timeout=15 -t 0 -q -O ${TEST_DATA_NAME}.tar.gz "${TEST_DATA_URL}" && tar xzf ${TEST_DATA_NAME}.tar.gz -C ~/data/; fi
     - docker load --input $HOME/docker/cache.tar || true
+    # Prepare MRIQCWebAPI
+    - pip install docker-compose
+    - pushd $HOME; git clone https://github.com/poldracklab/mriqcwebapi.git; pushd mriqcwebapi; git checkout 0.1.0; popd; popd
+    - docker-compose -f $HOME/mriqcwebapi/dockereve-master/docker-compose.yml build
+    - docker-compose -f $HOME/mriqcwebapi/dockereve-master/docker-compose.yml up -d
```
Comment (chrisfilo) [inline circle.yml]: I've used port 80 on circle successfully with Neurovault. We also use the following lines to get better log outputs and check if the server got up correctly:

```
- docker-compose --verbose up -d && sleep 50
- curl --retry 10 --retry-delay 15 -vkf http://localhost
```

**Comment by oesteban** [comment]
How is the query to retrieve all the current records in the database?

**Comment by chrisfilo** [comment]
`/T1w` and `/bold`. The results are paged.

**Comment by oesteban** [comment]
@chrisfilo, this is good to go. Do you want to take a look before merge?

**Pull request merged**